features/other_a/invite_queue.feature

```diff
@@ -35,6 +35,13 @@ Feature: Invite queue management
     When I follow "Get an Invitation"
     Then I should see "Request an Invitation"
 
+  Scenario: An admin can delete people from the queue
+    Given an invitation request for "invitee@example.org"
+      And I am logged in as an admin
+      And I go to the manage invite queue page
+      And I follow "Delete"
+    Then I should see "Request was removed from the queue."
```

Comment:
This is good! I'd just suggest adding one more thing to make sure we're ending up on the right page: `And I should be on the manage invite queue page` Doing that should cover the redirect on line 53 of the controller.